### PR TITLE
Feature/apollo graphqls schema

### PR DESCRIPTION
**Pull request by StylianosGakis**
Checkout the branch, delete the schema.json file and download the schema again with the new gradle.properties changes.

graphqls format gives some very nice navigation inside the IDE along with autocompletion for types. It really helps me since I am unfamiliar with basically all the queries/types right now. Maybe it's something we want to merge into develop for everyone to use from now on?

I don't know if you've looked into it before and decided against it due to some limitation or whatever.

### CODE REVIEW

File: app/src/engineering/java/com/hedvig/app/MockContractDetailViewModel.kt [hunk 1]
```diff
@@ -32,7 +32,7 @@ class MockContractDetailViewModel : ContractDetailViewModel() {
                     DocumentItems.Document(
                         titleRes = R.string.MY_DOCUMENTS_INSURANCE_TERMS,
                         subTitleRes = R.string.insurance_details_view_documents_insurance_letter_subtitle,
-                        uri = Uri.parse(it.termsAndConditions.url)
+                        uri = Uri.parse(it.termsAndConditions?.url)
```
Comment (hedvigoscar) [inline app/src/engineering/java/com/hedvig/app/MockContractDetailViewModel.kt:35]: These changes should not be required. Someone has snuck in a breaking change into the parent schema, and they will have to roll that breaking change back

**Review by hedvigoscar — APPROVED** [review]
Sensible. Let's roll back the `termsAndConditions`-things as they must be resolved upstream, however.

File: app/src/engineering/java/com/hedvig/app/MockContractDetailViewModel.kt (hunk 1 above)
Comment (StylianosGakis) [reply app/src/engineering/java/com/hedvig/app/MockContractDetailViewModel.kt:35]: Oopsies, I just wanted it to compile for now, will revert it back when it's rolled back then

**Comment by hugokallstrom** [comment]
Closing this since its fixed by another PR

**Pull request closed without merging**